src/app/item-popup/ItemStat.tsx

```diff
@@ -131,41 +164,52 @@ export function D1QualitySummaryStat({ item }: { item: D1Item }) {
   );
 }
 
+function getPlugHashesFromCategory(category: DimSocketCategory) {
+  return category.sockets
+    .map((socket) => {
+      return idx(socket, (socket) => socket.plug.plugItem.hash) || null;
+    })
+    .filter((hash) => !!hash);
```

Comment:
You don't need to change it, but in general we've preferred `Boolean(hash)` to `!!hash` - it's a bit more explicit. I swear we used to have a lint rule that enforced that...

You can also do `.filter(Boolean)` - again, just for future reference.